clarity-ext-scripts/clarity_ext_scripts/covid/pcr/example_result_file_rt_pcr.py

```diff
@@ -0,0 +1,89 @@
+import xlwt
+import collections
+import datetime
+import random
+from clarity_ext.extensions import GeneralExtension
+from clarity_ext_scripts.covid.parse_pcr import CT_HEADER
+
+
+class Extension(GeneralExtension):
+    def execute(self):
+        file_handle_name = "Result file"
+        timestamp = datetime.datetime.now().strftime("%y%m%dT%H%M%S")
```

Comment:
Added a timestamp to the second to ensure that we don't get that error about the file handle already being used.